kuma/wiki/migrations/0025_populate_document_uuid.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from uuid import uuid4
+
+from django.db import migrations
+
+
+def populate_uuids(apps, schema_editor):
+    """Populate Document.uuid, without bumping last modified."""
+    Document = apps.get_model('wiki', 'Document')
+    docs = Document.objects.filter(uuid__isnull=True)
+    for document_id in docs.values_list('id', flat=True).iterator():
+        Document.objects.filter(id=document_id).update(uuid=uuid4())
```

Comment:
Added ``.iterator()`` to reduce memory retained on admin node.

In reference to comment  https://github.com/mozilla/kuma/pull/3795#issuecomment-196392622 . Are you suggesting this migration should be removed from the "UUID step 2" PR #3794, and done manually in the shell instead?  If so, then it seems to make sense in this "UUID step 3" PR #3795, for developers etc. who didn't run the manual command.  If not, then I think it is in the right place.